app/services/rates_importer_service.rb

```diff
@@ -28,7 +28,7 @@ def import(row, rating_year, council)
         include_in_address_lookups: true,
         meta: row.to_s
       )
-    else property.update!(council: council, include_in_address_lookups: true, location: "#{location.gsub /"/, ''}".strip.titleize)
+    else property.update!(council: council, include_in_address_lookups: true, location: ("#{location.gsub /"/, ''} " + "#{suburb} " + town_city.to_s).strip.titleize)
```

Comment:
Should the test be updated accordingly?

Reply:
not in this case - these are one off data formats